apps/sync/js/crypto/stringconversion.js

```diff
@@ -23,6 +23,19 @@ const StringConversion = Object.freeze((() => {
       return byteArray;
     },
 
+    utf16StringToByteArray(str) {
+      if (typeof str != 'string') {
+        throw new Error('Not a string');
+      }
+      var string = unescape(encodeURIComponent(str)),
+          charList = string.split(''),
+          uintArray = [];
+      for (var i = 0; i < charList.length; i++) {
+          uintArray.push(charList[i].charCodeAt(0));
```

Comment:
http://stackoverflow.com/questions/14592364/utf-16-to-utf-8-conversion-in-javascript/14601808#14601808